### PR TITLE
Lua_api.txt: Fix documentation for facedir rotation

### PR DESCRIPTION
**Pull request by paramat**
Documentation was incorrect, axisdir is not facedir modulo 4.

### CODE REVIEW

**Comment by est31** [comment]
I don't see how the doc becomes more clear with this.

File: doc/lua_api.txt [hunk 1]
```diff
@@ -554,9 +554,11 @@ node definition:
     ^ The rotation of the node is stored in param2. Furnaces and chests are
       rotated this way. Can be made by using minetest.dir_to_facedir().
       Values range 0 - 23
-      facedir modulo 4 = axisdir
+      facedir / 4 = axis direction:
       0 = y+    1 = z+    2 = z-    3 = x+    4 = x-    5 = y-
-      facedir's two less significant bits are rotation around the axis
+      facedir modulo 4 = rotation around that axis
+      facedir 0, 1, 2, 3 = rotations around the +y axis
```
Comment (est31) [inline doc/lua_api.txt]: This is only an example for the general rule explained by lines 557, 558 and 559.

File: doc/lua_api.txt [hunk 2]
```diff
@@ -554,9 +554,11 @@ node definition:
...
+      facedir's two least significant bits are rotation around the axis
```
Comment (est31) [inline doc/lua_api.txt]: This is another way of putting line 559.

**Comment by est31** [comment]
Okay for me if the two lines are removed, but don't know how the actual format is, so can't +1 it.

**Comment by paramat** [comment]
Ok agreed, it's a bit verbose, i'll remove those 2 lines.

**Comment by paramat** [comment]
Updated.

**Comment by est31** [comment]
Yeah, that's fine. Thanks!

**Comment by paramat** [comment]
3f89bf4d7f241cebc118cccf2ac147ee6454216e

**Pull request closed without merging**